src/foam/nanos/auth/ChangePasswordView.js

```diff
@@ -50,22 +53,44 @@ foam.CLASS({
       padding-left: 2vh;
     }
     ^ .top-bar {
-      background: /*%PRIMARY1%*/ #202341;
       width: 100%;
       height: 8vh;
-      border-bottom: solid 1px #e2e2e3
+      border-bottom: solid 1px #e2e2e3;
     }
     ^ .top-bar img {
       height: 4vh;
-      padding-top: 1vh;
+      padding-top: 2vh;
+      display: block;
+      margin: 0 auto;
+    }
+    ^ .title-top {
+      font-size: 2.5em;
+      padding-top: 2vh;
+      font-weight: bold;
+      text-align: center;
+      margin-bottom: 0
+    }
+    ^ .subtitle {
+      color: #525455;
+      font-size: 16px;
+      line-height: 1.5;
+      margin-bottom: 15px;
```

Comment:
pixel? does this scale?

Reply:
I will take this into account when I work on refactoring.

Reply:
changed to relative values